### ISSUE TITLE
Make failing on AVIDump.cpp

### ISSUE THREAD

**Issue by lesurp** [FasterMelee/FasterMelee-installer#13]
When running the setup script, I encounter an error:

```
FasterMelee-installer/FasterMelee/Ishiiruka/Source/Core/VideoCommon/AVIDump.cpp: In static member function ‘static bool AVIDump::CreateVideoFile()’:
FasterMelee-installer/FasterMelee/Ishiiruka/Source/Core/VideoCommon/AVIDump.cpp:141:22: error: ‘avformat_alloc_output_context2’ was not declared in this scope
   s_dump_path.c_str()) < 0)
                      ^
make[2]: *** [Source/Core/VideoCommon/CMakeFiles/videocommon.dir/AVIDump.cpp.o] Error 1
make[1]: *** [Source/Core/VideoCommon/CMakeFiles/videocommon.dir/all] Error 2
make: *** [all] Error 2
```

Im looking on my side how to fix this...

System:
`Linux 3.16.0-4-amd64 #1 SMP Debian 3.16.43-2 (2017-04-30) x86_64 GNU/Linux`

**Comment by lesurp** [comment]
So this bugs is caused by the version of libavformat-dev used in Jessie.
An easy fix is installing the version available in jessie-backports:
`sudo apt-get install libavformat-dev -t jessie-backports
`
However apt-get refused to install it because of its dependencies, so I installed the package manually using `dpkg`, and then used `aptitude` **(not apt-get)** which offered me to upgrade a few other packages (available in the backports too).
Tl;dr: use Ubuntu/Arch, you'll save some time

**Comment by Ptomerty** [comment]
Good to know, I'll update the readme. Thanks!
